crypto/property/property.c

```diff
@@ -424,9 +427,10 @@ static void ossl_method_cache_flush_some(OSSL_METHOD_STORE *store)
     IMPL_CACHE_FLUSH state;
 
     state.nelem = 0;
-    state.store = store;
-    ossl_sa_ALGORITHM_doall_arg(store->algs, &impl_cache_flush_one_alg, &state);
+    if ((state.seed = (unsigned int)OPENSSL_rdtsc()) == 0)
```

Comment:
The cast can be removed. OPENSSL_rdtsc() returns a uint32_t:

https://github.com/openssl/openssl/blob/8c00f267b8df1a8c70eff8198de40aa561299e48/include/internal/cryptlib.h#L103

Reply:
Yeah, it can and done.